react/lib/util/api-client.ts

```diff
@@ -107,25 +108,23 @@ export default {
   getAddressBalance,
 };
 
-export const fiatCurrencies = ['USD', 'CAD'] as const;
 
-type fiatCurrenciesTuple = typeof fiatCurrencies; // readonly ['USD', 'CAD' ]
-export type fiatCurrency = fiatCurrenciesTuple[number]; // "USD" | "CAD" 
+type fiatCurrenciesTuple = typeof FIAT_CURRENCIES;
+export type fiatCurrency = fiatCurrenciesTuple[number];
```

Comment:
since we're at it, what about renaming this to UpperCamelCase, just as you renamed the constant to UPPER_SNAKE_CASE, following the conventions of not only constants but also types?